e2e/inputs.spec.ts

```diff
@@ -156,5 +156,5 @@ async function searchSplunk(page: Page, query: string) {
   await page.getByLabel("Search Button").click();
 
   await page.screenshot({ path: `playwright-screenshots/page-inputs-query-${query2file(query)}.png`, fullPage: true });
-  await expect(page.getByText("sourcetype").first()).toBeVisible();
```

Comment:
Why do you update the test?

Reply:
Agreed, so rollback the changes made on the tests and tried debugging on the playwright locally. and fixed it. :) Thanks!